docs/DeveloperGuide.md

```diff
@@ -669,9 +642,39 @@ Those without any stars are user stories that were considered but will not be im
 
       Use case ends.
 
+### **Use case: UCSH06 — See sample contacts in the system before starting to modify it**
+
+Preconditions: User has not added or edited contacts previously.
+
+**MSS**
+
+1.  User opens the application.
+2.  System shows a list of sample contacts.
+
+    Use case ends.
+
+### **Use case: UCSH07 — Reload sample contacts in the system**
+
+**MSS**
+
+1.  User requests to reload sample contacts into the system.
+2.  System deletes all current persons in the system and shows a list of sample contacts.
+
+    Use case ends.
+
+
+
+### **Use case: UCSH08 — See a list of all possible commands**
+
+**MSS**
+
+1.  User requests to see a list of all possible commands they can use in the system.
+2.  System shows a list of commands with their corresponding input format.
+
+    Use case ends.
 
+### **Use case: UCSH09 — Assign dietary requirement to contact**
```

Comment:
Are we still including this? If we are not planning to implement this feature in our final product then we shouldnt have a use case for it right

Reply:
This was in the original DeveloperGuide. I also thought of deleting it but wasn't sure as it was part of our Use Cases google sheets